File: app/src/main/java/com/example/music/hi/tab/common/IHiTabItem.java
package com.example.music.hi.tab.common;

import androidx.annotation.NonNull;
import androidx.annotation.Px;

public interface IHiTabItem<D> extends IHiTabLayout.OnTabSelectedListener<D> {
    void setHiTabInfo(@NonNull D data);

    void resetHeight(@Px int height);

}

File: app/src/main/java/com/example/music/hi/tab/common/IHiTabItem.kt
package com.example.music.hi.tab.common

import androidx.annotation.NonNull
import androidx.annotation.Px

interface IHiTabItem<D> : IHiTabLayout.OnTabSelectedListener<D>{
    fun setHiTabInfo(@NonNull info: D)
    fun resetHeight(@Px height: Int)
}